src/test/java/com/savvato/tribeapp/services/NotificationServiceImplTest.java

```diff
@@ -41,6 +50,37 @@ public NotificationService notificationService() {
 	@MockBean
 	NotificationRepository notificationRepository;
 
+
+	@Test
+	public void testCreateNotificationDTO() {
+		// Mock data
+		NotificationType mockType = new NotificationType();
```

Comment:
Use an actual NotificationType instance, like NotificationType.MEDIUM_PRIORITY, or whatever's defined.

Reply:
It looks like you went the other direction, and removed the testing of NotificationType. I believe that still needs to be included in the test, DTO, and associated objects.